PurrSoft/PurrSoft.Common/Helpful/SortAndPaginationUtils.cs

```diff
@@ -3,16 +3,16 @@
 public static class SortAndPaginationUtils
 {
     public static IQueryable<T> OrderByField<T>
-        (this IQueryable<T> source, string sortBy, string sortDirection)
+        (this IQueryable<T> source, string? sortBy, string? sortDirection)
     {
         if (string.IsNullOrWhiteSpace(sortBy))
         {
             return source;
         }
 
         var type = typeof(T);
-        var property = type.GetProperty(sortBy);
-        if (property == null)
+        var property = type.GetProperty(sortBy); // case-sensitive search
```

Comment:
I deleted that specific comment as it was redundant. The main issue I encountered with the SortAndPagination class was regarding the take parameter. Calling the method like this:
`overview = overview
	.SortAndPaginate(request.SortBy, request.SortOrder, request.Skip, request.Take);`
	
It would actually call the SortAndPaginate method with Take as 0 if the request didn't explicitly have Take as a parameter in its query. As such, I inserted a check to see whether the Take is larger than 0 inside the Paginate() method.

Aside from that I just added the `?` to some of the parameters in `OrderByField `for stronger type-checking, as we would already handle the null cases.